packages/shared/src/components/post/SquadPostContent.tsx

```diff
@@ -95,7 +95,7 @@ function SquadPostContent({
       >
         <div
           className={classNames(
-            'relative px-4 tablet:px-8',
+            'relative max-w-[42.5rem] px-4 tablet:px-8',
```

Comment:
I took `42.5rem` as that is the width of `pageMaxWidth`

Reply:
Hmm why do we need it?
Cant it just set to full?

Reply:
No, full doesn't work 😞

Reply:
The parent div already have a `!w-[42.5rem]` applied to it, but when it is a squad post, it also gets `flex-1` added, which then disregards the width of the element.

Reply:
Playing with it some more, setting `min-w-0` helped too, and it fixed another overflow issue on smaller screens.

Reply:
Lol we might have to start applying that mind w fix on *